### PR TITLE
Release 13.3.1

### PR DESCRIPTION
**Pull request by camscale**
* Added new Prometheus metric for created access requests. [#29991](https://github.com/gravitational/teleport/pull/29991)
* Added support for the web UI for automatically deploying a database service with ECS Fargate container when enrolling a new database. [#29978](https://github.com/gravitational/teleport/pull/29978)
* Added new Prometheus metrics to Kubernetes Access. [#29970](https://github.com/gravitational/teleport/pull/29970)
* Added ability to delete proxy resources with `tctl`. [#29903](https://github.com/gravitational/teleport/pull/29903)
* Added headless approval UI to Teleport Connect. [#28975](https://github.com/gravitational/teleport/pull/28975)
* Removed requiring team/channel inputs for mattermost plugin. [#30009](https://github.com/gravitational/teleport/pull/30009)
* Fixed change feed with PostgreSQL backend on Azure [#29911](https://github.com/gravitational/teleport/issues/29911) [#29975](https://github.com/gravitational/teleport/pull/29975)
* Fixed `tctl` to obey `--verbose` when formatting text tables. [#29870](https://github.com/gravitational/teleport/pull/29870)
* Updated OpenSSL to [3.0.10](https://github.com/openssl/openssl/blob/openssl-3.0.10/CHANGES.md#changes-between-309-and-3010-1-aug-2023). [#29908](https://github.com/gravitational/teleport/pull/29908)
* Updated Go to 1.20.7. [#29904](https://github.com/gravitational/teleport/pull/29904)
* Reduced logging level in PostgreSQL backend for improved performance. [#29847](https://github.com/gravitational/teleport/pull/29847)

### CODE REVIEW

**Comment by kimlisa** [comment]
> * Fixed Mattermost plugin to notify configured raw recipients. [#30009](https://github.com/gravitational/teleport/pull/30009)

can we change this too:

```
* Removed requiring inputs for team/channel for mattermost plugin. [#30009]
```

b/c the fix is for standalone plugin but no one is affected b/c the changes haven't gone in.

**Review by tcsc — APPROVED** [review]

**Review by fheinecke — APPROVED** [review]

**Comment by camscale** [comment]
I had to add `Fargate` to `docs/cspell.json` to pass the docs linter.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+* Added ability to delete proxy resources with `tctl`. [#29903](https://github.com/gravitational/teleport/pull/29903)
+* Added headless approval UI to Teleport Connect. [#28975](https://github.com/gravitational/teleport/pull/28975)
+* Removed requiring team/channel inputs for mattermost plugin. [#30009](https://github.com/gravitational/teleport/pull/30009)
+* Fixed change feed with PostgreSQL backend on Azure [#29911](https://github.com/gravitational/teleport/issues/29911) [#29975](https://github.com/gravitational/teleport/pull/29975)
+* Fixed `tctl` to obey `--verbose` when formatting text tables. [#29870](https://github.com/gravitational/teleport/pull/29870)
+* Updated OpenSSL to [3.0.10](https://github.com/openssl/openssl/blob/openssl-3.0.10/CHANGES.md#changes-between-309-and-3010-1-aug-2023). [#29908](https://github.com/gravitational/teleport/pull/29908)
+* Updated Go to 1.20.7. [#29904](https://github.com/gravitational/teleport/pull/29904)
+* Reduced logging level in PostgreSQL backend for improved performance. [#29847](https://github.com/gravitational/teleport/pull/29847)
```
Comment (espadolini) [inline CHANGELOG.md:15]: This is Postgres DB access, not the backend

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
 
+## 13.3.1 (08/04/23)
+
+* Added new Prometheus metric for created access requests. [#29991](https://github.com/gravitational/teleport/pull/29991)
+* Added support for the web UI for automatically deploying a database service with ECS Fargate container when enrolling a new database. [#29978](https://github.com/gravitational/teleport/pull/29978)
+* Added new Prometheus metrics to Kubernetes Access. [#29970](https://github.com/gravitational/teleport/pull/29970)
+* Added ability to delete proxy resources with `tctl`. [#29903](https://github.com/gravitational/teleport/pull/29903)
+* Added headless approval UI to Teleport Connect. [#28975](https://github.com/gravitational/teleport/pull/28975)
+* Removed requiring team/channel inputs for mattermost plugin. [#30009](https://github.com/gravitational/teleport/pull/30009)
+* Fixed change feed with PostgreSQL backend on Azure [#29911](https://github.com/gravitational/teleport/issues/29911) [#29975](https://github.com/gravitational/teleport/pull/29975)
```
Comment (espadolini) [inline CHANGELOG.md:11]: The error isn't Azure-specific, my bad for not providing a changelog entry in the PR :(

Comment (camscale) [reply CHANGELOG.md:11]: The bug report was for Azure. I didn't realise the fix was more broad. I've updated the changelog on the GitHub release.

File: CHANGELOG.md (hunk 1 above)
Comment (camscale) [reply CHANGELOG.md:15]: Ok, thanks. I've updated the changelog on the GitHub release

**Pull request merged**